src/main/java/graphql/nadel/engine/NadelExecutionStrategy.java

```diff
@@ -211,6 +223,19 @@ public NadelExecutionStrategy(List<Service> services,
         return resultNodes;
     }
 
+    private RootExecutionResultNode getSkippedServiceCallResult(NadelContext nadelContext, ExecutionStepInfo esi, ExecutionContext newExecutionContext, TransformationMetadata.NormalizedFieldAndError topLevelFieldError) {
+        HashMap<String, Object> stringObjectHashMap = new HashMap<>();
```

Comment:
we use as default always `LinkedHashMap` because it preserves the order of insertion.

Reply:
also: lets find a better name for the variable :-)

Reply:
Name suggestion: `nulledServiceData`